manifests/install_integration.pp

```diff
@@ -0,0 +1,20 @@
+define datadog_agent::install_integration (
+  Enum['present', 'absent'] $ensure           = 'present',
+  String                    $integration_name = undef,
+  String                    $version          = undef,
+){
+
+  include datadog_agent
+
+  if $ensure == 'present' {
+    $command = 'install'
+  } else {
+    $command = 'remove'
+  }
+
+  exec { "${command} ${integration_name}==${version}":
```

Comment:
How does the tool handle python2, python3 integration installations? I presume it installs for the python specified in the `datadog.yaml`.... Let's keep that in mind as a use case for the module, particularly when you check for an integration's presence.

Reply:
I think it does what you say, but in any case puppet should just assume the `integration` command does the right thing when multiple pythons are installed.

Reply:
This is true if we rely on the `integration` command entirely, but we first need to make sure the module isn't chatty before going for this approach.